index.bs

```diff
@@ -1306,24 +1760,68 @@ partial interface MLGraphBuilder {
+
+    : <dfn>activation</dfn>
+    ::
+        An {{MLActivation}} object. Specifies the optional activation function that immediately follows the normalization operation.
+</dl>
+
+<div>
     **Arguments:**
         - *input*: an {{MLOperand}}. The input N-D tensor.
-        - *mean*: an {{MLOperand}}. The 1-D tensor of the mean values of the input features across the batch whose length is equal to the size of the input dimension denoted by *options.axis*.
-        - *variance*: an {{MLOperand}}. The 1-D tensor of the variance values of the input features across the batch whose length is equal to the size of the input dimension denoted by *options.axis*.
-        - *options*: an optional {{MLBatchNormalizationOptions}}. The optional parameters of the operation.
-              - *scale*: an {{MLOperand}}. The 1-D tensor of the scaling values whose length is equal to the size of the input dimension denoted by *options.axis*.
-              - *bias*: an {{MLOperand}}. The 1-D tensor of the bias values whose length is equal to the size of the input dimension denoted by *options.axis*.
-              - *axis*: an {{unsigned long}} scalar. The index to the feature count dimension of the input shape for which the mean and variance values are. Its value must be in the range [0, N-1] where N is the rank of input tensor. When it's not specified, the default value is 1.
-              - *epsilon*: a {{float}} scalar. A small value to prevent computational error due to divide-by-zero. The default value is 0.00001 when not specified.
-              - *activation*: an {{MLActivation}}. The optional activation function that immediately follows the normalization operation.
+        - *mean*: an {{MLOperand}}. Specifies the 1-D tensor of the mean values of the input features across the batch whose length is equal to the size of the input dimension denoted by  {{MLBatchNormalizationOptions/axis}}.
+        - *variance*: an {{MLOperand}}. The 1-D tensor of the variance values of the input features across the batch whose length is equal to the size of the input dimension denoted by {{MLBatchNormalizationOptions/axis}}.
+        - *options*: an optional {{MLBatchNormalizationOptions}}. Specifies the optional parameters of the operation.
 
     **Returns:** an {{MLOperand}}. The batch-normalized N-D tensor of the same shape as the input tensor.
+</div>
 
-    When *input* is a 4-D tensor of the *"nchw"* or *"nhwc"* layout, *options.axis* should be set to 1 or 3 respectively. The axis value designates the feature or channel count dimension of the input tensor.
+<details open>
+  <summary>
+    The {{MLGraphBuilder/batchNormalization(input, mean, variance, options)}} method steps are:
+  </summary>
+  <div algorithm=batchnorm class=algorithm-steps>
+    1. [=Assert=]: the type of |input|, |mean| and |variance| is {{MLOperand}}.
+    1. [=Assert=]: the type of |mean| is
+    1. To validate |mean|, run the following substeps:
```

Comment:
This reads like it's defining an algorithm, not that the substeps are to be performed now. I'd drop this line and flatten the substeps into the main steps.